components/ContentTemplates/LinksTemplate.tsx

```diff
@@ -1,5 +1,392 @@
+							<span>
+								<ImNewTab size="1rem" aria-hidden="true" />
+							</span>{" "}
+							or a descriptive text e.g “(opens in a new tab)” and also include
+							a target attribute of “_blank” and rel=”noopener” to prevent
+							potential security risks.
+						</p>
+						<CodeBlock
+							codeSnippet={`<a href="https://www.accessibleweb.com" rel="noopener" target="_blank">
+    Visit Accessible Web
+    <span hidden id="new_tab">(opens in a new tab)</span>
+    <svg aria-hidden="true" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg" mirror-in-rtl="true" fill="#000000">
+    </svg>
+</a>`}
+							languageType={"html"}
+						/>
+						<p>
+							In this example, we use a <strong>{`${"<span>"}`}</strong> element
+							with the 'hidden' attribute and id="new_tab" with the text "(opens
+							in a new tab)", which won't be visible by default. The
+							aria-hidden="true" attribute in the SVG element ensures that the
+							SVG icon is not visible to screen readers. When screen readers
+							visit the link, they'll read it as{" "}
+							<strong>Visit Accessible Web. Opens in a new tab.</strong>
+						</p>
+					</li>
+				</ul>
+				<p>
+					Notice the difference between the hrefs. The href for internal link is
+					pointing to a page within this website - Buttons and the href for
+					external link is pointing to a file outside this website.
+				</p>
+			</TemplateSection>
+			<TemplateSection sectionName="linkText" title="Descriptive Link Text">
+				<p>
+					Link text is the text that describes a hyperlink on a webpage. It's
+					important to have descriptive link text to provide clear context for
+					screen readers and all users.
+				</p>
+				<CodeBlock
+					codeSnippet={`<a href="https://accessibleweb.dev">Visit Accessible Web</a>`}
+					languageType={"html"}
+				/>
+				<p>
+					Descriptive link text helps users understand the purpose of the link.
+					Avoid using vague phrases like "Click Here" or "Read More."
+				</p>
+				<p>
+					If you must use <strong>non-descriptive text</strong>, you can make it
+					more accessible with ARIA attributes for screen readers.
+				</p>
+				<CodeBlock
+					codeSnippet={`<a href="https://accessibleweb.dev" aria-label="Read more about Accessible Web">Read more</a>`}
+					languageType={"html"}
+				/>
+
+				<p>
+					<a
+						href="https://accessibleweb.dev"
+						aria-label="Read more about FreecodeCamp">
+						Read more
+					</a>
+				</p>
+
+				<p>
+					However, ARIA labels may not provide perfect accessibility. A better
+					solution is to use a <strong>{`${"<span>"}`}</strong> element within
+					the <strong>{`${"<a>"}`}</strong> element to provide descriptive text
+					that's accessible to screen readers but not visible to sighted users{" "}
+				</p>
+
+				<CodeBlock
+					codeSnippet={`<a href="https://www.freecodecamp.org">Read more
+	<span class="sr-only">Read more about FreeCodeCamp</span>
```

Comment:
On the buttons page the same class is called .visually-hidden as this technique is not always used only for screen readers. It would be great if you could update it here to use the same naming conventions